src/core/filesystem.cc

```diff
@@ -552,6 +552,7 @@ GCSFileSystem::FileModificationTime(const std::string& path, int64_t* mtime_ns)
   bool is_dir;
```

Comment:
Update filesystem.h. "for some filesystems a file or folder may not have a modification time, in that case m_time returns 0"

Reply:
Done